src/intrinsics.cpp

```diff
@@ -432,6 +450,19 @@ static jl_cgval_t emit_runtime_call(jl_codectx_t &ctx, JL_I::intrinsic f, const
     return mark_julia_type(ctx, r, true, (jl_value_t*)jl_any_type);
 }
 
+static jl_cgval_t emit_runtime_intrinsic(jl_codectx_t &ctx, jl_cgval_t *f, const jl_cgval_t *argv, size_t nargs)
+{
+    JL_I::intrinsic i = (JL_I::intrinsic)*(uint32_t*)f;
+    JuliaFunction *fptr = (JuliaFunction*)jl_intrinsic_table.fptrs[i];
+    Function *func = prepare_call(fptr);
+    Value **argvalues = (Value**)alloca(sizeof(Value*) * nargs);
+    for (size_t i = 0; i < nargs; ++i) {
+        argvalues[i] = boxed(ctx, argv[i]);
+    }
+    Value *r = ctx.builder.CreateCall(func, makeArrayRef(argvalues, nargs));
+    return mark_julia_type(ctx, r, true, (jl_value_t*)jl_any_type);
```

Comment:
```suggestion
    return emit_runtime_call(ctx, f, argv, nargs)
```
But shouldn't we have already handled this case, if it was possible?

Reply:
This is where I'm a bit hazy as well, @Keno, could you chime in?